src/utils.coffee

```diff
@@ -1,17 +1,30 @@
-# NOTE: using a custom build of lodash, to save space
-_ = require('../vendor/lodash.custom.min')
-uuid = require('uuid')
-sha1 = require('crypto-js/sha1')
-options = require('./options')
+import {v4} from 'uuid'
+import options from './options'
+
+sha1 = require('sha1')
 
 class Utils
-  @defaults: _.defaults
-  @keys: _.keys
-  @remove: _.remove
-  @omit: _.omit
+  @defaults: (obj, defaults) ->
```

Comment:
is there a simple way to test these against lodash's methods to make sure they are compatible?